src/website/page.gleam

```diff
@@ -395,7 +395,10 @@ pub fn news_post(post: news.NewsPost, ctx: site.Context) -> fs.File {
                 <> post.path
                 <> "')",
             ),
-            class("meta-button share-button"),
+            attr.data("tooltip-position", "right"),
+            attr.data("tooltip-trigger", "click"),
+            attr.aria_describedby("share-tooltip"),
```

Comment:
I don't think describedby is the correct attribute in this case. As it stands, when a user tabs to this button a screen reader will announce: "Share. Copied the post URL!".

Instead, remove the relationship between the button and the live region, initially render no text content in the live region, and on click set the live region's text content to "Copied the post URL."